File: android/app/src/main/java/com/phicdy/totoanticipation/view/activity/LicenseActivity.java
package com.phicdy.totoanticipation.view.activity;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.support.v7.widget.Toolbar;

import com.phicdy.totoanticipation.R;

public class LicenseActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_license);
        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);
        toolbar.setTitle(getTitle());
    }
}

File: android/app/src/main/java/com/phicdy/totoanticipation/view/activity/LicenseActivity.kt
package com.phicdy.totoanticipation.view.activity

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.support.v7.widget.Toolbar
import android.webkit.WebView
import com.phicdy.totoanticipation.R

class LicenseActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_license)
        val toolbar = findViewById<Toolbar>(R.id.toolbar)
        setSupportActionBar(toolbar)
        toolbar.title = title
        val webView = findViewById<WebView>(R.id.license_web_view)
        webView.loadUrl("file:///android_asset/license.html")
    }
}
